src/vaadin-button.html

```diff
@@ -166,6 +166,7 @@
           }
         }
 
+        /** @private */
```

Comment:
Nit: Should we prefer:
```js
/** @private */
```
vs
```js
/**
 * @private
 */
```
for single annotation JSdoc comments?

There is this above:
```js
        /**
         * @protected
         */
        disconnectedCallback() {
```
but the new one here is a oneliner.

Reply:
Not a blocker, but would be good to agree on a style for future consistency.

Reply:
I would prefer one-liners, there might be lots of them (especially in `vaadin-grid`).